### PR TITLE
Add minikube test

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Testing, this out so not ready :)

**Comment by shawn-hurley** [comment]
@ehelms I don't think I ever did I will get that done today though

**Comment by jwmatthews** [comment]
I think we owe @ehelms some beers at a future Beer Thursday.... :)

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+env:
+  - CHANGE_MINIKUBE_NONE_USER=true
+
+before_script:
+  # setup minikube
+  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+  - sudo minikube start --vm-driver=none --kubernetes-version=v1.9.0
+  - minikube update-context
+  - JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done
```
Comment (shawn-hurley) [inline .travis.yml]: Just for my understanding, this is waiting for all the nodes to be ready?

Comment (ehelms) [reply .travis.yml]: That's my understanding, I saw similar examples like this in the wild.

**Comment by shawn-hurley** [comment]
@fabianvf think that you could take a look?

**Pull request merged**